ReactiveObjC/RACStream.h

```diff
@@ -21,6 +21,12 @@ NS_ASSUME_NONNULL_BEGIN
 /// to be overridden.
 @interface RACStream<__covariant ValueType> : NSObject
 
+_Pragma("clang diagnostic push") \
+_Pragma("clang diagnostic ignored \"-Wstrict-prototypes\"") \
+typedef id _Nullable (^RACReduceBlock)();
+typedef ValueType _Nullable (^RACReduceBlock2)();
```

Comment:
`RACReduceBlock2` is not a great name IMO.

Let's either find a better name (`RACReduceValueTypeBlock`?) or remove it and just inline the necessary `#pragma`s in this case since it's only used in one spot.